src/Frontend/Modules/Location/Layout/Templates/Index.html.twig

```diff
@@ -25,9 +25,7 @@
 
             <div itemprop="name" class="hidden"><strong>{{ locationItem.title }}</strong></div>
             <address itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
-              <span itemprop="streetAddress">{{ locationItem.street }} {{ locationItem.number }}</span><br>
-              <span itemprop="postalCode">{{ locationItem.zip }}</span>
-              <span itemprop="addressLocality">{{ locationItem.city }}</span><br>
+              <span itemprop="streetAddress">{{ locationItem.address }}</span><br>
```

Comment:
Maybe we can add a setting where people can set the format they want to display with placeholders?
Something like this
```
[street] [number]
[postalCode] [locality]
[country]
```